compiler/testData/debug/stepping/IfTrueThenFalse.kt

```diff
@@ -0,0 +1,14 @@
+fun cond() = false
+
+fun box() {
+    if (cond())
+        cond()
+    else
+         false
+}
+
+fun main(args: Array<String>) {
```

Comment:
Idea: Similar to `LowLevelDebuggerTestBase`, maybe you can generate and write your own "main class" that simply calls `box()`? This way we can easily re-use existing box tests to create stepping tests.

Reply:
Was try to fix this, but getting some issue with class loader not found exception, for now I will just put a //FILE: main.kt for class to be load, and put MainKt in command line arguments, will address it once I figure out the issue.

Reply:
@max-kammerer Mike, do you prefer to include class name and function name in the line number verification data or just line number only?